scripts/metadataProvider/createPseudoModulesInfo.ts

```diff
@@ -39,28 +43,35 @@ async function transformFiles(sapui5Version: string) {
 		getPseudoModuleNames(),
 	]);
 
-	const {enums} = metadataProvider.getModel();
+	const {enums, namespaces: dataTypes} = metadataProvider.getModel();
 
-	const groupedEnums = Object.keys(enums)
-		.reduce((acc: Record<string, {enum: UI5Enum; export: string}[]>, enumKey: string) => {
+	const groupedEnums = Object.keys({...enums, ...dataTypes})
+		.reduce((acc: Record<string, {enum?: UI5Enum; dataType?: UI5Namespace; export: string}[]>, key: string) => {
 			// Filter only real pseudo modules i.e. defined within library.js files
-			if (!pseudoModuleNames[enumKey]) {
+			if (!pseudoModuleNames[key]) {
 				return acc;
 			}
 
-			const curEnum = enums[enumKey];
-
-			acc[curEnum.library] = acc[curEnum.library] ?? [];
-			acc[curEnum.library].push(
-				{enum: curEnum, export: pseudoModuleNames[enumKey]});
+			const curEnum = enums[key];
+			const curDataType = dataTypes[key];
```

Comment:
Hmmm, you're right. For the enums I wanted to reuse the `MetadataProvider`, but it lacks information about the data types. Now, as we eitherways need to read the `api.json`, it became obsolete and this could be indeed simplified.